jdbc/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Geode JDBC Connector Example
+
+The JDBC Connector allows Apache Geode to connect to external data sources with JDBC.
+
+![Apache Geode JDBC Connector](connector.svg)
+
+## Steps:
+
+1. Install MySQL: https://dev.mysql.com/downloads/
+
+If your MySQL installation does not include JDBC driver, 
+download it from https://dev.mysql.com/downloads/connector/j/
+
+2. Start MySQL server with `mysql.server start`. 
+Use `mysql` CLI to create database, table and populate the table:
+
+```
+create database geode_db;
+
+use geode_db;
+
+create table parent(id bigint, name varchar(100), income double);
+
+insert into parent values (2, 'Parent_2', 987654321.0);
+```
+
+3. Build this example's jar file `jdbc.jar` by running `../gradlew clean build`.
+
+The jar file `jdbc.jar` will be generated in `build/libs` directory.
+
+4. Add MySQL JDBC driver jar and `jdbc.jar` to `CLASSPATH`.
+
+e.g. 
+```
+export CLASSPATH=/path/to/mysql-connector-java-8.0.15.jar:/path/to/geode-examples/jdbc/build/libs/jdbc.jar
```

Comment:
Some other other examples just download dependencies through maven don't require the user to configure the classpath, for example luceneSpatial. It seems like the start script could at least automatically include the jdbc.jar?

Reply:
Good point. Will do.